tools/matchms/matchms_convert.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="matchms_convert" name="matchms convert" version="@TOOL_VERSION@+galaxy0" python_template_version="3.7">
+    <description>convert mass spectral library</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">matchms</requirement>
+    </requirements>
+
+    <command detect_errors="aggressive"><![CDATA[
+        python ${matchms_python_cli}
+    ]]></command>
+
+<configfiles>
+<configfile name="matchms_python_cli">
+from matchms.importing import load_from_msp, load_from_mgf
+from matchms.exporting import save_as_msp, save_as_mgf
+spectra = list(load_from_${spectral_library.ext}("${spectral_library}", ${harmonize_metadata}))
+save_as_${output_format}(spectra, "${converted_library}")
+</configfile>
+</configfiles>
+
+    <inputs>
+        <param label="Spectra file" name="spectral_library" type="data" format="msp,mgf,json" help="Mass spectra file to be filtered." />
```

Comment:
Does `json` input format work? The appropriate function (`load_from_json`) is not imported.

Reply:
It is now imported